pkg/ncproxy/nodenetsvc/v1/nodenetsvc.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package nodenetsvc.v1;
+package azure.nodenetworkservice;
```

Comment:
This is a breaking change. Should we version this one as well?

Reply:
I'm not sure if this needs to be versioned. The `azure.nodenetworkservice` api and corresponding server are produced by another team/repo and they are currently using the name `azure.nodenetworkservice` and I'm not sure if there's any plan for them to update that. I'm also not aware of another team producing a different variation of that api and server. What do you think?